lib/s3middleware/tagging.ts

```diff
@@ -100,7 +114,46 @@ function _validateTags(tags: Array<{ Key: string[], Value: string[] }>) {
     }
     // not repeating keys
     if (tags.length > Object.keys(tagsResult).length) {
-        return errors.InvalidTag.customizeDescription('Cannot provide ' +
+        return errors.InvalidTag.customizeDescription(
+            'Cannot provide multiple Tags with the same key'
+        );
+    }
+    return tagsResult;
+}
+
+/** validateTags - Validate tags, throwing an error if tags are invalid
+* @param tags - tags parsed from xml to be validated
+* @param tags[].Key - Name of the tag
+* @param tags[].Value - Value of the tag
```

Comment:
I don't think either of these are required as the tags type already describes them

Reply:
I removed them and updated the function jsdoc: https://github.com/scality/Arsenal/pull/1933/commits/9c5365cca8d2dfc6445792f2b23aa14586bd8cb4#diff-7aa7eeaed88643ea97d57a5f15ddc2e64c380190641c53bb0c3f64d67807a475R124